src/frontend/src/env/types/env-icrc-token.ts

```diff
@@ -29,3 +30,12 @@ export type EnvIcrcTokenMetadata = z.infer<typeof envIcrcTokenMetadata>;
 export type EnvIcrcToken = z.infer<typeof envIcrcToken>;
 
 export type EnvIcrcTokens = z.infer<typeof envIcrcTokens>;
+
+const envAdditionalIcrcTokens = z.record(envTokenSymbol, z.union([z.undefined(), envIcToken]));
+
+export type EnvAdditionalIcrcTokens = z.infer<typeof envAdditionalIcrcTokens>;
+
+export const envTokensAdditionalIcrc = z.object({
```

Comment:
envAdditionalIcrcTokens
EnvAdditionalIcrcTokens
envTokensAdditionalIcrc

Isn't it a bit inconsistent?
And the first is a schema.

Reply:
I tried to be aligned with https://github.com/dfinity/oisy-wallet/blob/main/src/frontend/src/env/types/env-token-ckerc20.ts

envCkErc20Tokens => envAdditionalIcrcTokens
EnvCkErc20Tokens => EnvAdditionalIcrcTokens
envTokensCkErc20 => envTokensAdditionalIcrc

Reply:
Ah thanks for letting know! I'll have a look (afterwards)